packages/inter-protocol/src/auction/scheduleMath.js

```diff
@@ -35,7 +37,7 @@ const subtract1 = relTime =>
  * their collateral).
  *
  * @param {Awaited<import('./params.js').AuctionParamManager>} params
- * @param {Timestamp} baseTime
+ * @param {TimestampRecord} baseTime
  * @returns {import('./scheduler.js').Schedule}
  */
 export const computeRoundTiming = (params, baseTime) => {
```

Comment:
The test that's changed in eb64643201b6ec seems to show that the wider `Timestamp` type does work here. While narrowing the type is a longer-term goal, the fact that a test had to be changed shows that it's a **breaking change**.

@Chris-Hibbert what do you think about narrowing this type at this point?

Reply:
I think this is movement forward. This particular change won't have wider impacts beyond the test, so it's fine to require compliance with the target state.